index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <h2>Duncan Njau</h2>
+        <img src="./images/hamburger.png" alt="image" />
+      </nav>
+    </header>
+    <section class="jumbotron">
```

Comment:
- [ ] I think you should consider using flexbox for the layout content. This gives you options to align the content nicely and neatly. Using flexbox will also make the section responsive 📱